### ISSUE TITLE
Normalize Custom Fields: Ability to transfer custom field definitions between custom-content types

### ISSUE THREAD

**Issue by GoogleCodeExporter** [viktorthang/wordpress-custom-content-type-manager#40]
```
What steps will reproduce the problem?
1. Manage the custom fields of any registered post_type
2. The only way to define a custom field is to define it from scratch.  Bummer.

What is the expected output? What do you see instead?
It would very useful to instead have an "Import Field Definition" link in the 
Custom Fields manager page (which lists all fields defined for a given 
post-type).  Clicking on that link would open up a "Field Definition Browser" 
that would list all registered post-types and the custom fields defined for 
each. 

I'm imagining each custom field would have a checkbox by it, and when the form 
is submitted, those field definitions would be copied into the current 
content-type.  Validation warnings would have to check for duplicate field 
names or the possibility of overwriting existing custom fields. 

To implement this, the CCTM.php file would need to be modified, specifically 
the man_controller function would need another action defined (each action is 
associated with an integer).  

```

Original issue reported on code.google.com by `fireproofsocks` on 3 Mar 2011 at 7:27

**Comment by GoogleCodeExporter** [comment]
```
The better approach may be to normalize the field definitions.
```

Original comment by `fireproofsocks` on 15 May 2011 at 1:47

**Comment by GoogleCodeExporter** [comment]
```
This will be handled by normalizing field definitions
```

Original comment by `fireproofsocks` on 3 Sep 2011 at 4:08
- Added labels: **Milestone-Release0.9.4**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fireproofsocks` on 15 Sep 2011 at 5:13
- Changed title: **Normalize Custom Fields: Ability to transfer custom field definitions between custom-content types**
- Changed state: **InProgress**

**Comment by GoogleCodeExporter** [comment]
```
This is ready and will be included in version 0.9.4.
```

Original comment by `fireproofsocks` on 16 Sep 2011 at 5:03
- Changed state: **Fixed**
